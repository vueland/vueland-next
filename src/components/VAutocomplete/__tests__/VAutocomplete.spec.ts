import { mount, VueWrapper } from '@vue/test-utils'
import { VAutocomplete } from '../index'

describe('VAutocomplete', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VAutocomplete, {
      ...options,
      global: {
        provide: {
          $options: null,
        },
      },
    })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
