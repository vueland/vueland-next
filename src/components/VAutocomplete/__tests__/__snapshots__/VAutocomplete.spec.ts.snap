// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VAutocomplete should mount component and match snapshot 1`] = `
<div class="v-input primary--text">
  <div class="v-input__text-field">
  </div>
  <div class="v-input__hints">
    <transition-stub>
    </transition-stub>
  </div>
</div>
`;
